views/default/input/color.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Elgg color input
+ * Displays a color input field
+ *
+ * @package Elgg
+ * @subpackage Core
```

Comment:
please remove all `@package` and `@subpackage` mentions, we don't use them correctly anyway or consistently